content/build-tool.md

```diff
@@ -118,7 +118,7 @@ CSS framework compatibility should be a primary concern when picking a pre-proce
 An important feature shared by all of the available CSS pre-processors is the ability to import files. This lets you split your CSS into smaller pieces, and provides a lot of the same benefits that you get from JavaScript modules:
 
 1. You can control the load order of files by encoding dependencies through imports, since the load order of CSS matters.
-2. You can create reusable CSS "modules" that just have variables and mixins and don't actually generate any CSS.
```

Comment:
I think a lot of `just` usage which has been removed in this PR had previously been _okay_, though I completely understand where you're coming from.  _Just_ letting you know my thinking. 😉

Reply:
:-)